src/domain/login/CompleteSSOLoginViewModel.js

```diff
@@ -24,15 +24,22 @@ export class CompleteSSOLoginViewModel extends ViewModel {
             loginToken,
             sessionContainer,
             attemptLogin,
-            showError,
         } = options;
         this._loginToken = loginToken;
         this._sessionContainer = sessionContainer;
         this._attemptLogin = attemptLogin;
-        this._showError = showError;
+        this._errorMessage = "";
         this.performSSOLoginCompletion();
     }
 
+    get errorMessage() { return this._errorMessage; }
+
+    _showError(message) {
+        this._errorMessage = message;
+        this.emitChange("errorMessage");
+        this._errorMessage = "";
```

Comment:
Yeah that wasn't very smart of me 🙂

Fixed for LoginViewModel and PasswordLoginViewModel but for CompleteSSOLoginViewModel I've removed `this._errorMessage = ""` since there's no UI currently for retrying a failed login using loginToken.